src/adapter/src/catalog.rs

```diff
@@ -1720,7 +1750,8 @@ pub struct CatalogEntry {
     oid: u32,
     name: QualifiedItemName,
     owner_id: RoleId,
-    privileges: Vec<MzAclItem>,
+    // Key is the role that granted the privilege, value is the privilege itself.
+    privileges: BTreeMap<RoleId, Vec<MzAclItem>>,
```

Comment:
Since this type is reused and redocumented so often, would it be easier to read if it were its own named type, either by alias or like a `struct(BTreeMap<..>)`?

Reply:
Updated, I created a type alias in `mz-sql::catalog`.